Formula/r.rb

```diff
@@ -74,6 +73,9 @@ def install
       ENV.append "LDFLAGS", "-L#{Formula[f].opt_lib}"
     end
 
+    # Avoid references to homebrew shims
+    args << "LD=#{Formula["binutils"].opt_bin}/ld"
```

Comment:
This line is problematic because `r` does not depend directly on `binutils`.
This line should be conditional on Linux.
```suggestion
    args << "LD=ld" unless OS.mac?
```